examples/benchmarks/bubblebench/simple-tree/CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# @fluid-example/bubblebench-editable-shared-tree
```

Comment:
This chnage log seems wrong. THats the wrong package name, and all those changes were not made to this package.

Reply:
oops it a typo, will fix it